lib/install/utils.go

```diff
@@ -381,19 +380,7 @@ type ExecResult struct {
 // FormatAbortError formats the specified error for output by the installer client.
 // Output will contain error message for err as well as any error it wraps.
 func FormatAbortError(err error) string {
-	switch err := err.(type) {
-	case trace.Error:
-		userMessage := trace.UserMessage(err)
-		if err.OrigError() != nil {
-			detail := trace.UserMessage(err.OrigError())
-			if detail != userMessage {
-				userMessage = fmt.Sprintf("%v (%v)", userMessage, detail)
-			}
-		}
-		return userMessage
-	default:
-		return trace.UserMessage(err)
-	}
+	return trace.UserMessage(err)
```

Comment:
Removed this as it would end up duplicating the error message here:
```go
userMessage = fmt.Sprintf("%v (%v)", userMessage, detail)
```
This [commit](https://github.com/gravitational/trace/pull/56/commits/1ad036c6ae5843db826b0b95a5aaba07bf8fbd65) in trace should render this unnecessary.